doc/htconfig.md

```diff
@@ -39,6 +39,9 @@ Example: To set the directory value please add this line to your .htconfig.php:
 * max_batch_queue - Default value is 1000.
 * max_processes_backend - Maximum number of concurrent database processes for background tasks. Default value is 5.
 * max_processes_frontend - Maximum number of concurrent database processes for foreground tasks. Default value is 20.
+* memcache (Boolean) - Use memcache. To use memcache the PECL extension "memcache" has to be installed and activated.
```

Comment:
Wouldn't it be confusing so that the beginning of a new list entry could be missed?